Sources/StreamChatUI/ChatChannelList/ChatChannelListVC.swift

```diff
@@ -17,7 +17,7 @@ open class ChatChannelListVC: _ViewController,
     /// The `ChatChannelListController` instance that provides channels data.
     public var controller: ChatChannelListController!
 
-    private var loadingPreviousMessages: Bool = false
+    private var paginatingChannels: Bool = false
```

Comment:
nit: it's preffered to name boolean variables starting with `is`
```suggestion
    private var isLoadingMoreChannels: Bool = false
```

Reply:
the same applies to channel vc: `loadingPreviousMessages -> isLoadingPreviousMessages`